playbooks/vars/sample-variables-sap-hypervisor-redhat-ocp-virt-preconfigure.yml

```diff
@@ -96,13 +97,23 @@ sap_hypervisor_node_preconfigure_cluster_config:
     password: xxxxx
     storage_driver: ontap-nas
     storage_prefix: ocpv_sap_
+    nfs_mount_options: nfsvers=3
+    export_policy: default
+    storageclass_name: sapstorage  # Name of the storage class (careful, there can only be one SC with this name)
+    default_storageclass: True     # Should this storage class be default?
 
   # CPU cores which will be reserved for kubernetes
   worker_kubernetes_reserved_cpus: "0,1"
 
   # Storage device used for host path provisioner as local storage.
   worker_localstorage_device: /dev/vdb
 
+  # Name of the storage class for locastorage
+  worker_localstorage_storageclass_name: sapstorage
+
+  # Should the local storage class be default?
+  worker_localstorage_default_storageclass: True
+
```

Comment:
another question, do we want to disable ipv4 bydefault for sap bridge? or is it a requirement to use only ipv6?

Reply:
No, we do not want to restrict to either ipv4 only or ipv6 only. Both should work, but I would focus on ipv4 in our testing.